### PR TITLE
Fix fs.Copy() to handle copying STI scripts

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
Should fix https://bugzilla.redhat.com/show_bug.cgi?id=1198888

@bparees @soltysh PTAL

File: pkg/util/fs.go [hunk 1]
```diff
@@ -76,12 +76,13 @@ func (h *fs) Exists(file string) bool {
...
-	info, err := os.Stat(sourcePath)
-	if err != nil {
+	if _, err := os.Stat(sourcePath); err != nil {
 		return err
 	}
 
-	if !info.IsDir() {
+	info, err := os.Stat(targetPath)
+
+	if err != nil || (info != nil && !info.IsDir()) {
```
Comment (soltysh) [inline pkg/util/fs.go]: I've just tested it, there's no need to check info for nillnes, either there's an error or you'll get that info.

**Comment by soltysh** [comment]
Other than this small nit, LGTM.

File: pkg/util/fs.go (hunk 1 above)
Comment (mfojtik) [inline pkg/util/fs.go]: I got panic if I don't check for nil here and the err is not nil

**Comment by soltysh** [comment]
[merge]

**Pull request merged**